client/files.go

```diff
@@ -98,12 +101,18 @@ func (c *Handler) handleCHMOD(params string) {
 }
 
 func (c *Handler) storeOrAppend(conn net.Conn, name string, append bool) (int64, error) {
-	flag := os.O_WRONLY
-	if append {
-		flag |= os.O_APPEND
+
+	if !append {
+		defaultPartSize := 1024 * 1024 * 4
```

Comment:
Why 4MB？

Reply:
https://github.com/yunify/qingstor-sdk-go/blob/master/client/upload/upload_client.go#L16
`qingstor-sdk-go/client/upload` limit `partSize` minimum to 4M.